tools/setup.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+import third_party
+from util import run, build_path, build_mode
+import os
+import distutils.spawn
+
+third_party.fix_symlinks()
+third_party.download_gn()
+third_party.download_clang()
+
+
+def get_gn_args():
+    out = []
+    if build_mode() == "release":
+        out += ["is_official_build=true"]
+    elif build_mode() == "debug":
+        pass
+    else:
+        print "Bad mode {}. Use 'release' or 'debug' (default)" % build_mode()
+        sys.exit(1)
+    if "DENO_BUILD_ARGS" in os.environ:
```

Comment:
How can I set debug specific args now (in particular, `symbol_level=2`) ?

Reply:
Use a different dir- I think we should have "debug" and "release" be defined configs for unambiguous error reporting.
```
export DENO_BUILD_PATH=C:/BertBuildDir/
./third_party/depot_tools/gn gen $DENO_BUILD_PATH --args="symbol_level=2"
./tools/build.py
```

Reply:
> ./third_party/depot_tools/gn gen $DENO_BUILD_PATH --args="symbol_level=2"

I wouldn't be so whiny if that worked today :)
I'll send a PR to add tools/gn.cmd and tools/ninja.cmd after this lands, and call it a day.

> The other question is how I can re-run setup.py while preserving any gn flags that I had previously set?

It seems that this is already working as I hoped.